autojenkins/jobs.py

```diff
@@ -295,19 +297,25 @@ def copy(self, jobname, copy_from='template'):
         params = {'name': jobname, 'mode': 'copy', 'from': copy_from}
         return self._build_post(NEWJOB, params=params)
 
-    def build(self, jobname, wait=False, grace=10):
+    def build(self, jobname, params=None, wait=False, grace=10):
         """
         Trigger Jenkins to build a job.
 
         :param wait:
             If ``True``, wait until job completes building before returning
+        :param params:
+            If params is a dictionary, append key, value pairs onto BUILD_W_PARAMS url
```

Comment:
I'm not comfortable with the fact that if it's not a dictionary, this is quietly ignored?
I'd prefer an exception.